src/lib/components/emptytable/Emptytable.component.js

```diff
@@ -0,0 +1,48 @@
+//@flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+import * as defaultTheme from "../../style/theme";
+import { getTheme } from "../../utils";
+
+type Props = {
+  children: Node | React.Node,
+};
+
+const EmptytableContainer = styled.tr`
+  width: 100%;
+  padding: ${defaultTheme.padding.large};
+  box-sizing: border-box;
+  display: table;
+  table-layout: fixed;
+
+  ${(props) => {
+    const brand = getTheme(props);
+    return css`
+      background: ${brand.primary};
+      border-top: 1px solid ${brand.border};
+    `;
+  }}
+`;
+
+const EmptytableContent = styled.td`
```

Comment:
We may have warning/issue in VolumeListTable in MetalK8s UI.
https://github.com/scality/metalk8s/blob/1fc1992657d54331d0b46b182dc7170685296ef3/ui/src/components/VolumeListTable.js#L347

Can you check that please?